notebooks/sycamore-tutorial-intermediate-etl.ipynb

```diff
@@ -0,0 +1,410 @@
+    "import sycamore\n",
+    "context = sycamore.init()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reading data\n",
+    "\n",
+    "Next, let's get some data to process. We will use a dataset of 73 NTSB airline report files in PDF format from a public bucket in Amazon S3. To read these PDF files, we can use the `read.binary` method.\n",
+    "\n",
+    "Sycamore reads these documents into a [DocSet](https://sycamore.readthedocs.io/en/stable/sycamore/get_started/concepts.html#docsets), which is a collection of documents and their associated elements. DocSets maintain this lineage as you break documents apart, and allow you to do transfrom the entire dataset versus needing to iterate document by document."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "paths = [\"s3://aryn-public/ntsb/\"]\n",
+    "\n",
+    "initial_docset = context.read.binary(paths = paths, binary_format = \"pdf\")\n",
+    "initial_docset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Hold on, hold on, did that do anything? Well, not really. Sycamore uses \"lazy exectution,\" meaning that it won't actually execute the processing steps in a pipeline until you execute a method that mandates it. This enables Sycamore to create efficient pipeline execution plans.\n",
+    "\n",
+    "In order to make Sycamore actually read the files, we need to use a DocSet method that forces execution. For example, `show`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "initial_docset.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`show` gets the first 10 (configurable by parameter) documents from the docset and prints them. So we can see 10 document objects, of type PDF, with some metadata in the `properties` object, and bytes in the `binary_representation` key. \n",
+    "\n",
+    "## Partitioning data\n",
+    "\n",
+    "We can now continue with our ETL pipeline. The next step is to partition the DocSet, which chunks each document into smaller elements using labeled bounding boxes. Each element could be a titles, table, table row and column, image, regular text, and more. This also extracts the actual information from the element (e.g. the information from the table), or does OCR. We use the [Aryn Partitioning Service (APS)](https://sycamore.readthedocs.io/en/stable/aryn_cloud/aryn_partitioning_service.html) to do this, and you can [get a free API key here](https://www.aryn.ai/get-started).\n",
+    "\n",
+    "In this example, we've set APS to extract table structures using a table structure recognition model, and to use OCR to get text from the document (but not to extract image contents). Using OCR takes a bit more time, but it can yield slightly higher quality text in some cases."
```

Comment:
I think `slightly higher quality text in some cases` is underselling a bit here. Maybe mention that it is also needed to get text out of image embedded documents?

Reply:
We cannot give explicit guidance to when OCR will be better than not using for documents that don't need traditional OCR. If you have a crisp way to describe that, happy to update it.